### PR TITLE
Random cleanup

### PR DESCRIPTION
**Pull request by blairconrad**
More minor cleaning up, just so we don't have to perform all these little changes after making _changes of consequence_.

### CODE REVIEW

**Comment by adamralph** [comment]
These changes have the consequence of warming my :heart:

**Comment by blairconrad** [comment]
Thanks @adamralph. There will probably be more.

File: src/FakeItEasy/Configuration/ArgumentCollection.cs [hunk 1]
```diff
@@ -66,10 +66,7 @@ public int Count
         /// <summary>
         ///   Gets the names of the arguments in the list.
         /// </summary>
-        public IEnumerable<string> ArgumentNames
-        {
-            get { return this.argumentNamesField; }
-        }
+        public IEnumerable<string> ArgumentNames => this.argumentNamesField;
```
Comment (thomaslevesque) [inline src/FakeItEasy/Configuration/ArgumentCollection.cs]: Shouldn't this be a read-only auto-property?

File: src/FakeItEasy/Core/FakeManager.cs [hunk 2]
```diff
@@ -86,15 +80,10 @@ public virtual IEnumerable<IFakeObjectCallRule> Rules
             get { return this.allUserRulesField.Select(x => x.Rule); }
         }
 
-        internal LinkedList<CallRuleMetadata> AllUserRules
-        {
-            get { return this.allUserRulesField; }
-        }
+        internal LinkedList<CallRuleMetadata> AllUserRules => this.allUserRulesField;
```
Comment (thomaslevesque) [inline src/FakeItEasy/Core/FakeManager.cs]: Same here

File: src/FakeItEasy/Core/TypeCatalogue.cs [hunk 3]
```diff
@@ -22,10 +22,7 @@ internal class TypeCatalogue : ITypeCatalogue
         /// <summary>
         /// Gets the <c>FakeItEasy</c> assembly.
         /// </summary>
-        public static Assembly FakeItEasyAssembly
-        {
-            get { return ExecutingAssembly; }
-        }
+        public static Assembly FakeItEasyAssembly => ExecutingAssembly;
```
Comment (thomaslevesque) [inline src/FakeItEasy/Core/TypeCatalogue.cs]: Same here

File: src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs [hunk 4]
```diff
@@ -203,10 +203,7 @@ public DelegateFakeObjectCall(Delegate instance, MethodInfo method, object[] arg
 
             public ArgumentCollection Arguments { get; }
 
-            public object FakedObject
-            {
-                get { return this.instance; }
-            }
+            public object FakedObject => this.instance;
```
Comment (thomaslevesque) [inline src/FakeItEasy/Creation/DelegateProxies/DelegateProxyGenerator.cs]: Here too

File: src/FakeItEasy/Creation/ProxyOptions.cs [hunk 5]
```diff
@@ -13,20 +13,11 @@ internal class ProxyOptions : IProxyOptions
...
-        {
-            get { return this.proxyConfigurationActions; }
-        }
+        public IEnumerable<Action<object>> ProxyConfigurationActions => this.proxyConfigurationActions;
 
-        public IEnumerable<CustomAttributeBuilder> AdditionalAttributes
-        {
-            get { return this.additionalAttributes; }
-        }
+        public IEnumerable<CustomAttributeBuilder> AdditionalAttributes => this.additionalAttributes;
```
Comment (thomaslevesque) [inline src/FakeItEasy/Creation/ProxyOptions.cs]: (not here, because internally they're lists)

File: src/FakeItEasy/Priority.cs [hunk 6]
```diff
@@ -35,10 +35,7 @@ private Priority(int value)
         /// <see cref="IFakeOptionsBuilder"/>s, and <see cref="IArgumentValueFormatter"/>s.
         /// </summary>
         /// <remarks>Equivalent to a Priority with value <c>0</c>.</remarks>
-        public static Priority Default
-        {
-            get { return DefaultInstance; }
-        }
+        public static Priority Default => DefaultInstance;
```
Comment (thomaslevesque) [inline src/FakeItEasy/Priority.cs]: Here too

File: src/FakeItEasy/Priority.cs [hunk 7]
```diff
@@ -55,10 +52,7 @@ public static Priority Default
         /// Just in case, though, all negative values have been reserved for use by FakeItEasy.
         /// </para>
         /// </remarks>
-        internal static Priority Internal
-        {
-            get { return InternalInstance; }
-        }
+        internal static Priority Internal => InternalInstance;
```
Comment (thomaslevesque) [inline src/FakeItEasy/Priority.cs]: and here

File: tests/FakeItEasy.Specs/LazySpecs.cs [hunk 8]
```diff
@@ -60,10 +60,7 @@ public class FooFactory : DummyFactory<IFoo>, IFoo
         {
             private static IFoo instance = new FooFactory();
 
-            public static IFoo Instance
-            {
-                get { return instance; }
-            }
+            public static IFoo Instance => instance;
```
Comment (thomaslevesque) [inline tests/FakeItEasy.Specs/LazySpecs.cs]: And here

**Comment by thomaslevesque** [comment]
Thanks @blairconrad. I had a few comments, but it seems I'm too late :wink:

**Comment by thomaslevesque** [comment]
All my comments are about cases where we have a read-only field and a read-only property that just returns the field; in those cases we might as well use a read-only auto-property. I could be mistaken in a few cases (I didn't always check the type of the field vs the type of the property).

**Comment by adamralph** [comment]
@thomaslevesque yes, I believe you're correct. I'm looking forward to the PR which simplifies those further! :wink:

**Comment by blairconrad** [comment]
:blush:

**Comment by thomaslevesque** [comment]
> I'm looking forward to the PR which simplifies those further! :wink:

Here it is #791 :wink: 

There were actually less occurrences of this than I thought

**Pull request merged**